docker-compose.yml

```diff
@@ -48,7 +48,7 @@ services:
         aliases:
          - wikibase-docker.svc
          - wikibase.svc
-         - portal.mardi4nfdi.${TLD}
+         #- portal.mardi4nfdi.${TLD}
```

Comment:
Please do not comment out code, you can just delete it. I am not sure if this does not break things otherwise.